igvm/kvm.py

```diff
@@ -260,7 +247,6 @@ def migrate_live(source, destination, vm, domain):
         migrate_cmd.format(
             domain=domain.name(),
             destination=destination.fqdn,
-            timeout=timeout,
```

Comment:
We still have 1GbE HVs so what do I do on them if the VM never finishes now? Also how has this been a problem?

Reply:
Okay let's remove it and see if anybody actually complains about it being gone.